### PR TITLE
feat: add nextProject option to handle monorepo

### PR DESCRIPTION
**Pull request by plevavas**

### CODE REVIEW

File: lib/cli/pack.ts [hunk 1]
```diff
@@ -84,12 +92,6 @@ export const packHandler = async ({ handlerPath, outputFolder, publicFolder, sta
 				path: '**/*',
 				ignore: ['**/node_modules/**', '*.zip'],
 			},
-			{
-				// Ensure hidden files are included.
-				isGlob: true,
-				cwd: standaloneFolder,
-				path: '.*/**/*',
-			},
```
Comment (plevavas) [inline lib/cli/pack.ts:92]: ℹ️  The .next folder is in a subfolder in the case of monorepo. So I add the option "dot" to true of the archiver to include also the hidden files.

**Comment by sladg** [comment]
How does standalone output work with monorepo? I was experiencing problems with Next not properly bundling dependencies for standalone output (aka. `.next/standalone/node_modules` folder was mostly empty). Do you have any monorepo example I might test on? :)

**Comment by plevavas** [comment]
> How does standalone output work with monorepo? I was experiencing problems with Next not properly bundling dependencies for standalone output (aka. `.next/standalone/node_modules` folder was mostly empty). Do you have any monorepo example I might test on? :)

After some tests, it works correctly. `.next/standalone/node_modules` is correctly filled. You can test with this repository : https://github.com/plevavas/monorepo

**Comment by sladg** [comment]
Hey! I went through your examples and tested it out.

To make it work, we need indeed `outputFileTracing` option, however, we also need `hoistingLimits: workspace` set as yarn option. See: https://github.com/vercel/next.js/issues/36386#issuecomment-1137665939

With that being said, I figured that we might ditch the option of `nextPath` and instead do it automatically (aka. search in output for `server.js` and make it work out of the box without adding parameters.

I will look at it during the day and try to make it work correctly for your usecase :)

**Comment by sladg** [comment]
Updated README with monorepo notes :)
Tested on your repo, all works fine now. No need for additional flag, lookup for nested output is automatic now.

**Comment by plevavas** [comment]
Thanks a lot for your reactivity 👍

**Pull request closed without merging**